src/applications/check-in/components/pages/Edit/Address.jsx

```diff
@@ -90,7 +92,7 @@ export default function Address(props) {
         if (!newValue) {
           setAddressError(prevState => ({
             ...prevState,
-            [fieldName]: `${event.target.label} is required`,
+            [fieldName]: `${event.target.label} ${t('is-required')}`,
```

Comment:
How is this handling the `{event.target.label}` translation?

Reply:
that is a good catch, we probably need to hit our appConstants with translation.

Reply:
yeah good call, looking at this 🙂

Reply:
good point, yeah I think that could wait then

Reply:
I made a note here: https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/38260